components/playground/playground.go

```diff
@@ -359,7 +360,36 @@ func (p *Playground) handleScaleIn(w io.Writer, pid int) error {
 	return nil
 }
 
-func (p *Playground) sanitizeConfig(boot instance.Config, cfg *instance.Config) {
+func (p *Playground) absDir(dir string) (string, error) {
```

Comment:
Can you add a test for this function? I'm not sure if dir like `$HOME/a/b/c` works

Reply:
OK